### PR TITLE
Update UI

### PR DESCRIPTION
**Pull request by rui-han-crh**
## UI

- Inspect: `inspect INDEX <OR> NAME` to swap the inspect panel information
- ShowNotes: `showNotes` slides the notes panel into view if hidden, otherwise, no-op
- HideNotes: `hideNotes` hides the notes panel if visible, otherwise, no-op
- Loan history is viewed from most recent on top to earliest records below
- If any list is filtered, a filter icon will show to inform the user that they are not seeing the full list
- Panes anchored to a ratio of the window size
- ResultsDisplay is now detached from the initial scene. Click on the command box to show the display and press `ESC` or click on anywhere else that is not the command text field to hide the results display

- ~~**BUG**: A small middle area of the person cards cannot be selected for some reason~~

image

### CODE REVIEW

**Review by Pinran-J — COMMENTED** [review]
Tested JAR file, and the following functions: 
- [x] All notes functions
- [x] showNotes/hideNotes/inspect

The small bug with clicking on person card seems to be due to this red line
image

File: src/main/resources/view/PersonListCard.fxml [hunk 1]
```diff
@@ -2,36 +2,37 @@
...
+  <VBox alignment="CENTER" minHeight="30">
+    <VBox alignment="CENTER_LEFT" VBox.vgrow="ALWAYS" prefHeight="80">
+      <VBox VBox.vgrow="ALWAYS">
+        <Label fx:id="id" styleClass="cell_big_label" text="\$id" wrapText="true"/>
+        <VBox VBox.vgrow="ALWAYS" alignment="CENTER">
+          <Label fx:id="name" textAlignment="CENTER" text="\$first" styleClass="cell_big_label" wrapText="true"/>
+        </VBox>
+      </VBox>
+    </VBox>
+    <Separator styleClass="separator-card" maxWidth="80">
```
Comment (Pinran-J) [inline src/main/resources/view/PersonListCard.fxml:22]: Bug mentioned might have been caused by this seperator.

Comment (rui-han-crh) [reply src/main/resources/view/PersonListCard.fxml]: Okay, fixed

**Review by Pinran-J — APPROVED** [review]
LGTM, good job!

**Review by Neethesh26 — APPROVED** [review]
LGTM, good job !

**Pull request merged**